<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectROIDialog</class>
 <widget class="QDialog" name="SelectROIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame, QDialog {
background-color: white;
}

#btnROIFrame {
border-top: 1px solid #b0b0b0;
}

QLabel, #roiTitlelbl {
color: black;
font: 20px;
font-weight: bold;
}

/* Style PushButton */

QPushButton {
border: none;
border-bottom: 1px solid #b0b0b0;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
background-color: #FFFFFF;
color: #000000;
font-weight: bold;
font: 15px;
}

QPushButton::hover {
color: #2f9aff;
border-bottom: 4px solid #2f9aff;
}

QPushButton::pressed {
background-color: 2f9aff;
border-radius: 10px;
border-bottom: 2px solid #2f9aff;
background-color: #c6c6c6
}

#btnFinish {
border: none;
border-bottom-left-radius: none;
border-top-left-radius: 10px;
border-top: 2px solid #b0b0b0;
border-left: 2px solid #b0b0b0;
}

#btnFinish::hover {
border-top: 4px solid #ff373a;
border-left: 4px solid #ff373a;
}

#btnFinish::pressed {
border-top: 2px solid #ff373a;
border-left: 2px solid #ff373a;
}

/* Style TableView and Header */

QTableView {
background-color: #FFFFFF;
color: black;
border: 1px solid  #b0b0b0;
border-radius: 10px;
}

QHeaderView::section {
background-color:  #d8f8ff;
color: black;
border: 1px solid  #b0b0b0;
}

/* Style stacked widget */
#stackROILabel {
border: 1px #b0b0b0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="roiTitlelbl">
     <property name="text">
      <string>SELECT ROI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bodyFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="leftFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackROILabel"/>
         </item>
         <item>
          <widget class="QFrame" name="btnROIFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="btnBack">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/LeftArrow/icons/arrow-left.png</normaloff>:/LeftArrow/icons/arrow-left.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="btnClear">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/Clean/icons/clean.png</normaloff>:/Clean/icons/clean.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QPushButton" name="btnNext">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/RightArrow/icons/arrow-right.png</normaloff>:/RightArrow/icons/arrow-right.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="btnFinish">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/Escape/icons/escape.png</normaloff>:/Escape/icons/escape.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
